src/pages/index.astro

```diff
@@ -54,13 +54,13 @@ deadlines = deadlines
         />
 
         <Title level={3} className="font-bold text-3xl mb-4 mt-12 lg:mt-auto">
-          On-site Volunteer
+          Stills and behind the scenes
         </Title>
 
         <ButtonLink
-          href="/volunteers"
+          href="/Photos"
         >
-          Join Us in Shaping EuroPython 2024
+          Peruse the conference and social event photos
```

Comment:
I think the title should be more obvious and short, e.g. 2024 Photos. You want people from most backgrounds to immediately recognise what this is about and click on it :) The button text can be longer and descriptive like what you have or  e.g. Conference stills and behind-the-scenes moments

Reply:
I did some adjustments. Better? 
I don't want to have too much text in the button and I want to have an action indicated in the text as well. So even now I think the text is too long.

Reply:
Looks OK to me, but I agree that it shorter would be ideal. Ideas: "Moments of Togetherness" or "Captured Connections".

Reply:
i did use a combination of our short DM chat for the button text. Thank you.